contentcuration/contentcuration/frontend/channelEdit/components/edit/CompletionOptions.vue

```diff
@@ -175,15 +175,10 @@
       hideDurationDropdown() {
         // named "hide" instead of "show" because "show" is the default behavior
         if (this.value) {
-          if (this.kind === ContentKindsNames.H5P) {
-            return true;
-          }
           return (
-            this.currentCompletionDropdown === CompletionDropdownMap.reference ||
-            (this.value.model === CompletionCriteriaModels.REFERENCE &&
-              !this.currentCompletionDropdown) ||
-            //should be hidden if model is reference and we're getting this from the BE
-            this.currentCompletionDropdown === CompletionDropdownMap.determinedByResource
+            this.value.model == CompletionCriteriaModels.REFERENCE ||
+            this.value.model == CompletionCriteriaModels.DETERMINED_BY_RESOURCE ||
+            this.kind === ContentKindsNames.H5P
```

Comment:
Not sure I understand why we're doing this, as H5P can have a duration completion criterion set?

Reply:
This was me misreading the figma spec -- updated